exercise/static/exercise/duplicate_check.js

```diff
@@ -0,0 +1,67 @@
+function openDuplicateModalOrSubmit(exercise, hashes, hash, submitCallback) {
+  const isDuplicate = hashes.includes(hash);
+  if (isDuplicate) {
+    // Set the number of the existing duplicate submission
+    $("#duplicate-submission-modal").find("[data-dup-submission]").text(hashes.reverse().indexOf(hash) + 1);
+    // Unbind previous event handlers and bind a new one
+    $("#duplicate-submission-modal-button").unbind('click');
```

Comment:
> As of jQuery 3.0, .unbind() has been deprecated. It was superseded by the [.off()](https://api.jquery.com/off/) method since jQuery 1.7, so its use was already discouraged.

https://api.jquery.com/unbind/

Let's use the new API `off` instead of `unbind`.